behaviours/rant_patrol.py

```diff
@@ -6,6 +6,7 @@
 
 RANT_REGEX = re.compile("^[^ ]*(rant|vent) *:", flags=re.IGNORECASE)
 FORMAT_REGEX = re.compile("^(anti-|co-)*(rant|vent): ", flags=re.IGNORECASE)
+STEM_REG = re.compile("^(anti-|co-)*(rant|vent):", flags=re.IGNORECASE)
```

Comment:
What is the point of this? Remove the trailing space from the regex above instead.

Reply:
LoC incentives strike again